### PR TITLE
[DOC IMP] Translate Smart Energy tutorial and custom cluster to Chinese

### PR DESCRIPTION
**Pull request by Jim-tech**
**Description about the pull request**
- Test-Result-Custom-Cluster-Commands.png

**Review**
@MarkDing @ChengYuan-CY

### CODE REVIEW

File: zigbee/Zigbee-Custom-Clusters-CN.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+在抓包中，您可以找到有关这两个命令的详细信息。
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Custom-Clusters/Test-Result-Extended-Commands.png">
+</div>
+</br>
+
+## 3. 添加自定义 Cluster
+### 3.1. 定义自定义 Cluster
+要添加自定义群集，我们需要首先在 xml 中定义它。我们有一个示例，它是 `app\zcl` 目录中的 `sample-extensions.xml`。
```
Comment (ChengYuan-CY) [inline zigbee/Zigbee-Custom-Clusters-CN.md:256]: 群集 -> Cluster

File: zigbee/Zigbee-Custom-Clusters-CN.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+</div>
+</br>
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Custom-Clusters/Custom-Cluster-Commands.png">
+</div>
+</br>
+
+### 3.3. 测试
+创建启用自定义Cluster `Hello World` 的服务器端的项目，同时创建一个启用自定义Cluster `Hello World` 的客户端的另一个项目。在服务器项目上，检查要测试的回调 `Command One`。
```
Comment (ChengYuan-CY) [inline zigbee/Zigbee-Custom-Clusters-CN.md:341]: 检查要测试的回调 -> 勾选要测试的回调

File: zigbee/Zigbee-Custom-Clusters-CN.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+### 2.3. 测试
+创建一个启用了 `Level Control` 服务器端的项目，以及启用了 `Level Control` 客户端的另一个项目。在服务器项目中，勾选回调 `Sample Mfg Specific Cmd` 和 `Sample Mfg Specific Cmd With Args` 进行测试。
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Custom-Clusters/Test-Extended-Command-Handling.png">
+</div>
+</br>
+
+在文件 `<projectname>_callbacks.c` 中, 添加下面两个回调函数的实现:
+```
```
Comment (ChengYuan-CY) [inline zigbee/Zigbee-Custom-Clusters-CN.md:184]: change the "```" to "```c", the C source code will have a better looking

File: zigbee/Zigbee-Smart-Energy-Tutorial-CN.md [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+14.  开始编译.
+&nbsp; 
+
+### 3.2. NCP 模式
+1. 打开 Simplicity Studio 菜单`File` -> `New` -> `Project`，启动创建工程的向导；
+2. 选择 `Silicon Labs AppBuilder Project`，然后点击`Next`；
+3. 选择 `Customizable network coprocessor (NCP) Applications`， 然后点击`Next`；
+4. 选择 `NCP UART HW (Hardware Flow Control)`， 然后点击`Next`；
+5. 命名你的工程, 例如 `zse_ncp`， 然后点击`Next`；
+6. 在接下来的窗口(`Project Setup`)中, 选择开发板型号, 并且设置编译器未`IAR ARM`，然后点击`Finish`；
```
Comment (ChengYuan-CY) [inline zigbee/Zigbee-Smart-Energy-Tutorial-CN.md:145]: 未 -> 为

**Comment by Jim-tech** [comment]
Thanks @ChengYuan-CY . I have made a little change according to your comment.

**Pull request merged**